runtime/oti/bcverify.h

```diff
@@ -169,10 +169,29 @@ typedef struct J9BCVAlloc {
 	temp1 = POP; \
 	inconsistentStack |= (temp1 == BCV_BASE_TYPE_TOP);
 
-/* Must be 2 singles or a correctly ordered wide pair */
-#define POP_TOS_PAIR( temp1, temp2 )	\
+#define IS_WIDE_TYPE(temp)	\
+	(J9_ARE_ALL_BITS_SET(temp, BCV_BASE_TYPE_LONG) | J9_ARE_ALL_BITS_SET(temp, BCV_BASE_TYPE_DOUBLE))
```

Comment:
Nitpick - please wrap `temp` in parenthesis so it will be `(temp)`.

Reply:
Updated.